torchvision/prototype/datasets/_builtin/gtsrb.py

```diff
@@ -17,15 +15,31 @@
 )
 from torchvision.prototype.features import Label, BoundingBox, EncodedImage
 
+from .._api import register_dataset, register_info
 
-class GTSRB(Dataset):
-    def _make_info(self) -> DatasetInfo:
-        return DatasetInfo(
-            "gtsrb",
-            homepage="https://benchmark.ini.rub.de",
-            categories=[f"{label:05d}" for label in range(43)],
-            valid_options=dict(split=("train", "test")),
-        )
+NAME = "gtsrb"
+
+
+@register_info(NAME)
+def _info() -> Dict[str, Any]:
+    return dict(
+        categories=[f"{label:05d}" for label in range(43)],
```

Comment:
Given that this dataset is also has no human readable categories, we could also drop them here. Let's resolve this in https://github.com/pytorch/vision/pull/5745#discussion_r843688607 and follow the same approach.